scoutcloud/scoutcloud/src/server/settings.rs

```diff
@@ -0,0 +1,63 @@
+use blockscout_service_launcher::{
+    launcher::{ConfigSettings, MetricsSettings, ServerSettings},
+    tracing::{JaegerSettings, TracingSettings},
+};
+use serde::Deserialize;
+
+#[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
+#[serde(deny_unknown_fields)]
+pub struct Settings {
+    #[serde(default)]
+    pub server: ServerSettings,
+    #[serde(default)]
+    pub metrics: MetricsSettings,
+    #[serde(default)]
+    pub tracing: TracingSettings,
+    #[serde(default)]
+    pub jaeger: JaegerSettings,
+    pub database: DatabaseSettings,
+    pub github: GithubSettings,
+}
+
+impl ConfigSettings for Settings {
+    const SERVICE_NAME: &'static str = "SCOUTCLOUD";
+}
+
+#[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
+#[serde(deny_unknown_fields)]
+pub struct DatabaseSettings {
+    pub url: String,
+    #[serde(default)]
+    pub create_database: bool,
+    #[serde(default)]
+    pub run_migrations: bool,
+}
```

Comment:
We have `DatabaseSettings` defined in blockscout-service-launcher. Maybe use that one?